Formula/wxpython.rb

```diff
@@ -34,24 +33,34 @@ class Wxpython < Formula
     depends_on "gtk+3"
   end
 
+  resource "numpy" do
+    url "https://files.pythonhosted.org/packages/c5/63/a48648ebc57711348420670bb074998f79828291f68aebfff1642be212ec/numpy-1.19.4.zip"
+    sha256 "141ec3a3300ab89c7f2b0775289954d193cc8edb621ea05f99db9cb181530512"
+  end
+
   resource "Pillow" do
-    url "https://files.pythonhosted.org/packages/3e/02/b09732ca4b14405ff159c470a612979acfc6e8645dc32f83ea0129709f7a/Pillow-7.2.0.tar.gz"
-    sha256 "97f9e7953a77d5a70f49b9a48da7776dc51e9b738151b22dacf101641594a626"
+    url "https://files.pythonhosted.org/packages/2b/06/93bf1626ef36815010e971a5ce90f49919d84ab5d2fa310329f843a74bc1/Pillow-8.0.1.tar.gz"
+    sha256 "11c5c6e9b02c9dac08af04f093eb5a2f84857df70a7d4a6a6ad461aca803fb9e"
   end
 
   resource "six" do
     url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
     sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
   end
 
+  resource "wxPython" do
+    url "https://files.pythonhosted.org/packages/b0/4d/80d65c37ee60a479d338d27a2895fb15bbba27a3e6bb5b6d72bb28246e99/wxPython-4.1.1.tar.gz"
+    sha256 "00e5e3180ac7f2852f342ad341d57c44e7e4326de0b550b9a5c4a8361b6c3528"
+  end
+
   def install
     # Fix build of included wxwidgets:
     # https://github.com/wxWidgets/Phoenix/issues/1247
     # https://github.com/Homebrew/homebrew-core/pull/58988
     inreplace "buildtools/build_wxwidgets.py" do |s|
       s.gsub! "#wxpy_configure_opts.append(\"--enable-monolithic\")",
               "wxpy_configure_opts.append(\"--disable-precomp-headers\")"
-      s.gsub! "--with-macosx-version-min=10.9",
+      s.gsub! "--with-macosx-version-min=10.10",
```

Comment:
The `--with-macosx-version-min` fix has been committed upstream (but not the other one)

Reply:
Got it.